packages/flutter/lib/src/widgets/tap_and_drag_gestures.dart

```diff
@@ -689,9 +684,9 @@ mixin _TapStatusTrackerMixin on OneSequenceGestureRecognizer {
 ///
 /// When competing against [DragGestureRecognizer], if the pointer does not move a sufficient
 /// global distance to be considered a drag, the recognizers will tie in the arena. If the
-/// pointer does travel enough distance then the [TapAndDragGestureRecognizer] will lose because
-/// the [DragGestureRecognizer] will declare self-victory when the drag threshold is met.
-class TapAndDragGestureRecognizer extends OneSequenceGestureRecognizer with _TapStatusTrackerMixin {
+/// pointer does travel enough distance then the recognizer that entered the arena
```

Comment:
How does this translate to widget tree of RawGestureDetector? i.e. if I nested `TapAndDragGestureRecognizer` which one would win

Reply:
If you nested them and the pointer has moved past the drag threshold (considered a drag), then the first `TapAndDragGestureRecognizer` instance to receive the `PointerEvent` will win the arena because it declares victory immediately. I think the first one to receive the event is the child since hit testing propagates from inner -> outer from my understanding.

If it has not moved past the drag threshold (considered a tap), then the first recognizer to enter the arena will win (i.e. they both tie and the gesture arena will `sweep` so first member of arena wins).

This is inline with `TapGestureRecognizer` and `DragGestureRecognizer`.

Reply:
Sounds good, can you add that into the doc? I think it would be clearer than the current doc

Reply:
Sure thing!